source/install/packstack.html.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Packstack quickstart
+title: Packstack
 category: documentation
 authors: apevec, dneary, garrett, jasonbrooks, jlibosva, mmagr, pixelbeat, pmyers,
   rbowen, gbraad, cbrown2
 wiki_category: Documentation
-wiki_title: Quickstart
+wiki_title: Packstack
```

Comment:
I checked the Ruby magic and `wiki_title` is used in `config.rb`. Fortunately this is not needed anymore and will save us some build time :-)

So could you finish this PR, removing this part of the change, and I'm going to create another one for the cleanup ?